### DISCUSSION TITLE
❗ Add a stepper component to Orbit

### DISCUSSION THREAD

**Discussion by alexasselin008** [gsoft-inc/sg-orbit#1122] [New components Ideas]
## Description

In some of our applications, we have the following component: 

![image](https://user-images.githubusercontent.com/38871812/194601997-0ee1ac1e-a65b-41f2-9fde-6fdf0f258c9a.png)
![image](https://user-images.githubusercontent.com/38871812/194613364-ff0bcd8e-fe5e-4fbc-82d5-da689e801db6.png)

**Comment by kcharen** [comment]
Reference: Ant design system has a [dot stepper version ](https://ant.design/components/steps/#components-steps-demo-progress-dot)

Questions: 
- Will we have error state for a step? 
- Will we set process values (they may be also called as states)? E.g., wait, process, finish, error. 
These are the values in Ant Design system. Adding this for reference.

**Comment by patricklafrance** [comment]
I suggest moving this to a Github discussion.
